src/npc.h

```diff
@@ -381,6 +381,8 @@ enum talk_topic_enum {
     TALK_LOOK_AT,
     TALK_OPINION,
 
+    TALK_CAMP_OVERSEER,
```

Comment:
The comment above says:
```
 // DO NOT USE! This is old, use strings as talk topic instead, e.g. "TALK_AGREE_FOLLOW" instead of
 // TALK_AGREE_FOLLOW. There is also convert_talk_topic which can convert the enumeration values to
// the new string values (used to load old saves).
```